src/e2e-test/java/io/cdap/plugin/cloudsqlpostgresql/stepsdesign/CloudSqlPostGreSql.java

```diff
@@ -55,10 +78,9 @@ public void targetIsCloudSQLPostGreSQL() throws InterruptedException {
         CdfCloudSqlPostGreSqlActions.selectCloudSQLPostGreSQLSink();
     }
 
-    @Then("Open CloudSQLPostGreSQL Properties")
-    public void openCloudSQLPostGreSQLProperties() throws InterruptedException {
-        CdfCloudSqlPostGreSqlActions.clickCloudSqlPostGreSqlProperties();
-        SeleniumHelper.waitElementIsVisible(CdfCloudSqlPostGreSqlLocators.validateBtn, 10);
+    @When("Target is BigQuery")
+    public void targetIsBigQuery() {
+        CdfStudioActions.sinkBigQuery();
     }
 
     @Then("Validate Connector properties")
```

Comment:
name of the method is not matching step description

Reply:
done